Foundation/NumberFormatter.swift

```diff
@@ -1,26 +1,14 @@
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Copyright (c) 2014 - 2016, 2019 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
-// See http://swift.org/LICENSE.txt for license information
-// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
 import CoreFoundation
 
-#if os(macOS) || os(iOS)
```

Comment:
These are necessary to build on Darwin, because Darwin's clang provides `__attribute__((objc_fixed_enum))` and Linux clang doesn't.

Reply:
Those constants aren't actually used in `NumberFormatter.swift`. Additionally I built/tested in Xcode and it worked fine.